### PR TITLE
Timestamp logging

### PR DESCRIPTION
**Pull request by matheuspf**

### CODE REVIEW

File: examples/classifier.c [hunk 1]
```diff
@@ -205,13 +205,37 @@ void train_classifier_valid(char *datacfg, char *cfgfile, char *weightfile, int
...
+
+    sprintf(summary_file_path, "%s/%s", time_stamp_dir, params.hyper_param_file);
+    FILE* summary_file_ts = fopen(summary_file_path, "w+");
 
     sprintf(log_file_path, "%s/%s", backup_directory, params.log_file);
     FILE* log_file = fopen(log_file_path, weightfile ? "a+" : "w+");
 
-    sprintf(log_file_path, "%s/%s", backup_directory, params.hyper_param_file);
-    FILE* summary_file = fopen(log_file_path, weightfile ? "a+" : "w+");
+    sprintf(summary_file_path, "%s/%s", backup_directory, params.hyper_param_file);
```
Comment (csantosbh) [inline examples/classifier.c]: Shouldn't the hyperparams file be placed in the session directory?

File: examples/classifier.c [hunk 2]
```diff
@@ -205,13 +205,37 @@ void train_classifier_valid(char *datacfg, char *cfgfile, char *weightfile, int
...
+    FILE* summary_file = fopen(summary_file_path, "w+");
```
Comment (csantosbh) [inline examples/classifier.c:223]: Is there a reason to have two summary files placed in distinct places?

**Review by csantosbh — APPROVED** [review]
Approved! (But take a look at my comment before merging)

File: examples/classifier.c (hunk 2 above)
Comment (matheuspf) [reply examples/classifier.c:223]: The first summary file (summary_file_ts) points to the new directory, created with time stamp, and saves all the information about the current run. The second points to the current running summary, so we always have fresh info about the training execution, even if it stops somehow.

**Pull request merged**